doc/userguide/configuration/multi-tenant.rst

```diff
@@ -152,7 +153,7 @@ Unix Socket
 Registration
 ~~~~~~~~~~~~
 
-register-tenant <id> <yaml>
+``register-tenant <id> <yaml>``
```

Comment:
So, given that multiple devices can have the same tenant id. if 2 devices had tenant id 1, i'd register them as
```
register-tenant 1 1st.yaml
register-tenant 1 2nd.yaml
```
?

Reply:
Yes,  but you'd also have to register the handler so something like this would work
```
register-tenant 1 1st.yaml
register-tenant-handler 1 vlan 3933
```

Reply:
Should we add an example like that, then? :)